sources/cql.l

```diff
@@ -218,13 +218,14 @@ static bool_t cql_already_processed_file(CSTR str) {
 
 stop [^A-Z_0-9]
 sp [ \t]+
+nl \r\n?|\n\r?
```

Comment:
This matches \r or \r\n or \n\r or \n  Two of which are not standard line endings and one is only used on DOS?  What's going on here?

Reply:
I think \n\r is just wrong.

Reply:
```
nl \r\n|\n
```

works for me